frontend/components/CameraLoadWidget.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect, useState } from 'react';
+
+import { exec } from '../util/exec';
+
+import Bar from '../util/Bar';
+
+const CameraLoadWidget = props => {
+  let [load, setLoad] = useState([0, 0, 0]);
+  useEffect(() => {
```

Comment:
Please cleanup the `setInterval` otherwise the `uptime` command is executed even if the load page is not viewed, and even multiple times, if one opens the load page mulitple times.

Reply:
Taken care in upcoming PR